src/catchup/ApplyCheckpointWork.cpp

```diff
@@ -323,12 +323,6 @@ ApplyCheckpointWork::onRun()
         mConditionalWork->startWork(wakeSelfUpCallback());
         return State::WORK_RUNNING;
     }
-    catch (InvariantDoesNotHold&)
-    {
-        // already displayed e.what()
-        CLOG_ERROR(History, "Replay failed");
-        throw;
-    }
```

Comment:
you can stop including the related header

Reply:
that said... I think the intent of the throw here was to ensure that we get a core dump in this kind of situation

Reply:
never mind: this exception cannot happen anymore as we queue up an `ApplyLedgerWork`, this was missed when we made the `ApplyLedgerWork` change